### ISSUE TITLE
Visualize Test Predictions after Training

### ISSUE THREAD

**Issue by cmhhelgeson** [trekhleb/micrograd-ts#2]
Currently DemoMLPTraining visualizes the final predictions of the model based on the training data alongside a graph showing the loss over time. However, there is no visualization of how the model performs on a range of test data after it has been trained.

**Comment by trekhleb** [comment]
Fixed by https://github.com/trekhleb/micrograd-ts/pull/1
